File: Back/src/ProEventos.Domain/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace ProEventos.Domain
{
    public class Event
    {
        public int Id { get; set; }
        public string Place { get; set; }
        public DateTime? EventDate { get; set; }
        public string Theme { get; set; }
        public int AmountPeople { get; set; }
        public string ImageURL { get; set; }
        public string Phone { get; set; }
        public string Email { get; set; }
        public IEnumerable<Batch> Batches {get; set;}
        public IEnumerable<SocialMedia> SocialMedias {get; set;}
        public IEnumerable<SpeakerEvent> SpeakersEvents { get; set; }




    }
}

File: Front/ProEventos-App/src/app/models/Event.ts
import { Batch } from "./Batch";
import { SocialMedia } from "./SocialMedia";
import { Speaker } from "./Speaker";

export interface Event {
  id : number;
  place : string;
  eventDate? : Date;
  theme : string;
  amountPeople : number;
  imageURL : string;
  phone : string;
  email : string;
  batches : Batch[];
  socialMedias : SocialMedia[];
  speakersEvents : Speaker[];
}
